SYCL/DiscardEvents/discard_events_usm_ooo_queue.cpp

```diff
@@ -12,6 +12,10 @@
 // The test checks that the last parameter is not `nullptr` for all PI calls
 // that should discard events.
 // {{0|0000000000000000}} is required for various output on Linux and Windows.
+// NOTE: piextUSMEnqueuePrefetch in the CUDA backend may return a warning
+//       result on Windows with error-code -996
+//       (PI_ERROR_PLUGIN_SPECIFIC_ERROR). Since it is a warning it is safe to

```

Comment:
Technically, the PI_ERROR_PLUGIN_SPECIFIC_ERROR can mean both error and warning, it is determined by a subsequent call to piPluginGetLastError. But I think it is OK for this test not to bother about it.